src/UI.ts

```diff
@@ -629,6 +631,19 @@ class UI {
     }
     videoEl.addEventListener('play', this.onVideoElPlay)
 
+    this.onVolumeChange = (event: any) => {
+      Logger.log('ima volume change ')
+      videoEl.muted = convertLocalStorageIntegerToBoolean('StroeerVideoplayerMuted')
+      videoEl.volume = convertLocalStorageStringToNumber('StroeerVideoplayerVolume')
+      const volumeInPercent = videoEl.volume * 100
+      volumeLevelBubble.style.bottom = 'calc(' + String(volumeInPercent) + '% - 4px)'
+      volumeLevelBubble.style.top = 'auto'
```

Comment:
Could this setting be moved to where the initial settings of the volume bar are set? This seems to have a constant value which does not need to be re-set